.circleci/config.yml

```diff
@@ -6,73 +6,125 @@ jobs:
     working_directory: /mnt/crate
     steps:
       - checkout
-      - run: git submodule sync
-      - run: git submodule update --init
       - restore_cache:
           keys:
-            - cargo-v2-{{ checksum "Cargo.toml" }}-
-            - cargo-v2-
+            - cargo-cache-{{ checksum "Cargo.toml" }}-{{ checksum "Cargo.lock" }}
+            - cargo-cache-{{ checksum "Cargo.toml" }}
+            - cargo-cache-
```

Comment:
Why did you rename the prefix? The old `cargo-v2-*` entries are still valid for this cache, and the versioning is a useful practice for forcing a fresh cache when we decide we don't need some previously cached directories.

Reply:
re-renamed